package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "weatherapp",
+  "version": "1.0.0",
+  "description": "This app will help you know the weather condition in any city you want.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
* [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

```
"dev": "webpack --mode development",
    "build": "webpack --mode production",
```